src/client/public/index.script.ts

```diff
@@ -11,6 +13,31 @@ import { Strings } from '../app/strings';
     document.getElementById('choose-your-host').textContent = strings.HtmlPageStrings.chooseYourHost;
     document.getElementById('choose-your-host').style.visibility = 'visible';
 
+    let ribbons = document.getElementsByClassName('ribbon');
+    for (let i = 0; i < ribbons.length; i++) {
```

Comment:
Tried this already: due to the nature of `HTMLCollection` , `forEach` or normal array iteration won't work. The only property exposed seems to be `length`, which is why I'm using it:

https://stackoverflow.com/questions/22754315/for-loop-for-htmlcollection-elements